cloudflare/resource_cloudflare_account_member.go

```diff
@@ -0,0 +1,114 @@
+package cloudflare
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"strings"
+
+	cloudflare "github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+// This needs to be hooked up to the provider schema instead.
+var cloudflareOrgID = os.Getenv("CLOUDFLARE_ORG_ID")
+
+func resourceCloudflareAccountMember() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceCloudflareAccountMemberCreate,
+		Read:   resourceCloudflareAccountMemberRead,
+		Update: resourceCloudflareAccountMemberUpdate,
+		Delete: resourceCloudflareAccountMemberDelete,
+
+		SchemaVersion: 0,
+		Schema: map[string]*schema.Schema{
+			"email_address": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"role_ids": {
```

Comment:
Decided to use IDs over Names here as the name _could_ change but the IDs are pretty unique and won't trigger a bunch of extra updates.